tests/trainer/logging_/test_logger_connector.py

```diff
@@ -470,3 +471,37 @@ def training_step(self, *args, **kwargs):
     )
     with pytest.warns(UserWarning, match="The progress bar already tracks a metric with the .* 'loss'"):
         trainer.fit(model)
+
+
+@pytest.mark.parametrize("auto_add_dataloader_idx", [False, True])
+def test_auto_add_dataloader_idx(tmpdir, auto_add_dataloader_idx):
+    """ test that auto_add_dataloader_idx argument works """
+
+    class TestModel(EvalModelTemplate):
```

Comment:
Can you update the test to use BoringModel instead?

Reply:
Done :]